### DISCUSSION TITLE
HideFirmware feature test

### DISCUSSION THREAD

**Discussion by ghost** [sandboxie-plus/Sandboxie#4008] [Q&A Feedback]
In Sandboxie Classic I just add "HideFirmwareInfo=y" to a box, but see no "FalseFirmware" registry entry amywhere.

**Comment by love-code-yeyixiao** [comment]
This feature does not create a registry per se so far, at this stage you will need to create it yourself and give it a value, later we may give it the ability to automatically create a random registry.

**Comment by ghost** [comment]
Can you provide an example of software that accesses firmware info, can run in Sandboxie and what false value works with it? I just want to test the feature. 

Can it spoof system specs for Steam or other gaming services?

**Reply by love-code-yeyixiao** [reply]
I don't see a typical software like this,but you can try some software which generate machine codes according to different machines.Any value type is OK,no matter it is String ,DWORD or BYTES.
To work better,I advise you to test with BlockAccessWMI template as well.

Only applications which use NtQuerySystemInformation can be spoofed.
